src/Database/Expression/TupleComparison.php

```diff
@@ -32,7 +32,7 @@ class TupleComparison extends ComparisonExpression
      * @var array<string|null>
      * @psalm-suppress NonInvariantDocblockPropertyType
      */
-    protected $_type;
+    protected array $types;
```

Comment:
ComparisonExpression::_type is `?string` and cannot be re-declared as array here.